### PR TITLE
Add movement update callback

### PR DESCRIPTION
**Pull request by jorbascrumps**

### CODE REVIEW

**Comment by jdnichollsc** [comment]
Hi Christopher,
Let me check your code at weekend!
Thanks, Nicholls

**Comment by jdnichollsc** [comment]
Hi my friend,
Can you add an example please? Check the [current examples](https://github.com/jdnichollsc/Phaser-Kinetic-Scrolling-Plugin/tree/master/examples)
Thank you for your contribution, Nicholls

**Comment by jorbascrumps** [comment]
Hi @jdnichollsc! I've added an example and updated the README. Sorry for the delay. Please let me know if anything else is required.

**Comment by jdnichollsc** [comment]
Hi @iamchristopher, sorry for the delay, I'm very busy, let me check your amazing work this weekend! 👍

**Comment by jorbascrumps** [comment]
Hi @jdnichollsc, have you had a chance to look at this?

**Comment by jdnichollsc** [comment]
Hi bro! Sorry, I changed the job and I was very busy, let me check your changes and I want to test on Mobile :)

**Comment by jdnichollsc** [comment]
@iamchristopher Hi bro, can you check my comments and update the version to **1.0.5** please?
Also, can you modify the README to add you as collaborator with your info? 👍 
Let me know what you think, Nicholls

**Comment by jorbascrumps** [comment]
Hi @jdnichollsc, which comments are you referring to?

**Comment by jdnichollsc** [comment]
@iamchristopher check my comments in your pull request bro 👍 https://github.com/jdnichollsc/Phaser-Kinetic-Scrolling-Plugin/pull/9/files/f8dcff462d0a84c1d3aff07efdab15801bc7f81a#diff-041033fc2aaa3c520a376de14f350de5

**Comment by jorbascrumps** [comment]
@jdnichollsc There are no comments showing up for me.

File: src/phaser-kinetic-scrolling-plugin.js [hunk 1]
```diff
@@ -259,12 +277,30 @@
...
 
             this.velocityWheelY += delta;
+
+            if (typeof this.settings.onUpdate === 'function') {
+                var deltaY = 0;
+                if (this.game.camera.y > 0 && this.game.camera.y + this.game.camera.height < this.game.camera.bounds.height) {
+                    deltaY = delta;
+                }
+
+                this.settings.onUpdate(deltaY);
```
Comment (jdnichollsc) [inline src/phaser-kinetic-scrolling-plugin.js]: @iamchristopher Hi bro, can you change this line to accept this pull request?
Example:
```
this.settings.onUpdate(0, deltaY);
```

File: src/phaser-kinetic-scrolling-plugin.js [hunk 2]
```diff
@@ -259,12 +277,30 @@
...
 
             this.velocityWheelX += delta;
+
+            if (typeof this.settings.onUpdate === 'function') {
+                var deltaX = 0;
+                if (this.game.camera.x > 0 && this.game.camera.x + this.game.camera.width < this.game.camera.bounds.width) {
+                    deltaX = delta;
+                }
+
+                this.settings.onUpdate(deltaX);
```
Comment (jdnichollsc) [inline src/phaser-kinetic-scrolling-plugin.js]: @iamchristopher Hi bro, can you change this line to accept this pull request?
Example:
```
this.settings.onUpdate(deltaX, 0);
```

**Comment by jorbascrumps** [comment]
@jdnichollsc Thanks for the feedback! It was quite valuable. I've made the requested changes. Let me know if you need anything else before merging let me know!

**Comment by jdnichollsc** [comment]
@iamchristopher Thanks bro for your excellent contribution and sorry for the delay! 🍻

**Pull request merged**